### PR TITLE
Replace both CAM_IP and CAM_PW by more generic CAM_URL

### PR DESCRIPTION
**Pull request by romainreignier**
This allows to use RTSP cameras with other path or username.
But because we use OpenCV, one can also use a webcam like /dev/video0
But the device /dev/video0 needs to be added to docker-compose.yml

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -15,4 +15,6 @@ services:
         # You need to update this to absolute path of this dir on your machine
         source: C:\Users\caleb\projects\github-sleepy-baby\BabySleepCoach\
         target: "${APP_DIR}"
-    container_name: baby-sleep-coach
```
Comment (calebolson123) [inline docker-compose.yml:18]: any reason to remove container name?

File: main.py [hunk 2]
```diff
@@ -995,13 +995,11 @@ def _build_cors_preflight_response():
 
 def receive(producer_q):
     print("Start receiving frames.")
-    cam_ip = os.environ['CAM_IP']
```
Comment (calebolson123) [inline main.py:998]: Good stuff, this is cleaner, thanks

File: docker-compose.yml (hunk 1 above)
Comment (calebolson123) [reply docker-compose.yml:18]: I guess it's not used/referenced anywhere else, dont really care. Going to merge while I'm here. Thanks for doing this @romainreignier

Comment (romainreignier) [reply docker-compose.yml:18]: Oops, this is an error, sorry.

Comment (calebolson123) [reply docker-compose.yml:18]: haha no worries, shouldn't break anything

**Pull request merged**